static/js/components/Header.js

```diff
@@ -7,7 +7,7 @@ class Header extends React.Component {
     return <div id="header">
       <div className="header-logo">
         <a href="/" alt="MIT Micromasters">
```

Comment:
I don't think `a` tags should have `alt`: http://stackoverflow.com/questions/14850187/is-it-correct-to-use-alt-tag-for-an-anchor-link